src/components/MoneyRequestHeader.tsx

```diff
@@ -132,6 +132,7 @@ function MoneyRequestHeader({report, parentReportAction, policy, onBackButtonPre
 
     const shouldShowBrokenConnectionViolation = shouldShowBrokenConnectionViolationTransactionUtils(parentReport, policy, transactionViolations);
     const isReportSubmitter = isCurrentUserSubmitter(chatIOUReport);
+    const isParentReportDM = isDM(parentReport);
```

Comment:
👋 Hey, coming from https://github.com/Expensify/App/issues/75520
This has caused an issue with educational modal(s) appearing twice (`HOLD` modal on the first hold, `HOLD&REJECT` on the next one)
We should have used `chatIOUReport` instead of `parentReport` (same as in `MoneyReportHeader`)